packages/flutter/lib/src/material/raised_button.dart

```diff
@@ -63,7 +65,7 @@ class RaisedButton extends StatelessWidget {
     this.disabledElevation = 0.0,
     this.padding,
     this.shape,
-    this.clipBehavior = Clip.none,
+    this.clipBehavior = ui.defaultClipBehavior, // ignore: deprecated_member_use
```

Comment:
You provide the same default value for clipBehavior in RawMaterialButton, so there's no need to specify it here.

Reply:
Because clipBehavior is defined as a default parameter on RawMaterialButton  and we unconditionally pass clipBehavior from all parent widgets, it will end up as null unless we specify it at every level.

Reply:
I don't think we should include a clipBehavior parameter here (and etc).